packages/appstore/webpack.config.js

```diff
@@ -139,24 +139,7 @@ module.exports = function (env) {
                 },
                 {
                     test: /\.svg$/,
-                    issuer: /\/packages\/wallets\/.*(\/)?.*.scss/,
-                    exclude: /node_modules/,
-                    include: /public\//,
-                    type: 'asset/resource',
-                    generator: {
-                        filename: 'appstore/wallets/public/[name].[contenthash][ext]',
-                    },
-                },
-                {
-                    test: /\.svg$/,
-                    issuer: /\/packages\/wallets\/.*(\/)?.*.tsx/,
-                    exclude: /node_modules/,
-                    include: /public\//,
-                    use: svg_loaders,
-                },
-                {
-                    test: /\.svg$/,
-                    exclude: [/node_modules|public\//],
```

Comment:
Removed the extra config added previously for `@deriv/wallet` SVG use case, Now this needs to be handled in `@deriv/core`.